application/authoriser/handler.py

```diff
@@ -23,19 +26,48 @@ def authorize_api_request(event, context) -> dict:
         )
     return response
 
-def check_authorisation_token(token_hash_sent: str, noauth: bool) -> bool:
+
+def fetch_secret_token(current_window: str) -> str:
+    global cached
```

Comment:
Why is this global?

Reply:
afaik only way to access and share it with the test outside of a class - please correct me if I'm wrong..

Reply:
Should be accessible via the instantiated class, or `self.cached` with the class

Reply:
ok @forjayssake so cached is mainly for the tests to know if caching was used or not. As this is not a class it cannot have an internal property. Any suggestions otherwise?